blah.py

```diff
@@ -1,2 +1,3 @@
 import numpy
 my_arr = np.array([1, 2, 3, 5, 'fish'])
+print('this is some nice code')
```

Comment:
```suggestion
print('this is some nice code!')
```

This could use an exclamation point